plugins/reporters/static-html/src/main/kotlin/StaticHtmlReporter.kt

```diff
@@ -150,7 +150,7 @@ class StaticHtmlReporter : Reporter {
                     index(reportTableModel)
 
                     reportTableModel.ruleViolations?.let {
-                        evaluatorTable(it)
+                        ruleViolationTable(it)
```

Comment:
Commit message: s/the the/on the/

Reply:
Should this say "ruleViolationsTable" (plural) to align with the "ruleViolations" variable?

Reply:
> Should this say "ruleViolationsTable" (plural) to align with the "ruleViolations" variable?

I've aligned this with the already existing analog `issueTable`. (As I also like plural form more, I'd fix this in a following PR).